src/main/java/org/embulk/input/zendesk/ZendeskInputPlugin.java

```diff
@@ -199,6 +200,17 @@ public void cleanup(final TaskSource taskSource, final Schema schema, final int
     public TaskReport run(final TaskSource taskSource, final Schema schema, final int taskIndex, final PageOutput output)
     {
         final PluginTask task = taskSource.loadTask(PluginTask.class);
+
+        if (Exec.isPreview()) {
+            checkValidTimeRangeThrowException(task);
+        }
+        else {
+            if (!isValidTimeRange(task)) {
+                logger.info("It is not a valid time range, end time: '" + task.getEndTime().get() + "'. Skip this run");
```

Comment:
Please explain why it is invalid time range, with the current message, user need to guess the reason because end time is in the future.

Reply:
Should update the message as :
The end time, '" + task.getEndTime().get() + "', is greater than the current time. No records will be imported. Data will be imported in next runs

Reply:
updated